windows.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+const path = require('path');
+
+const fastcall = require('fastcall');
+const ref = fastcall.ref;
+const wchar = require('ref-wchar');
+
+// Required by QueryFullProcessImageName
+// https://msdn.microsoft.com/en-us/library/windows/desktop/ms684880(v=vs.85).aspx
+const PROCESS_QUERY_LIMITED_INFORMATION = 0x1000;
+
+// Used by GetWindowLong
+// https://msdn.microsoft.com/en-us/library/windows/desktop/ms633584(v=vs.85).aspx
+const GWL_ID = -12;
```

Comment:
I understand that we have to hard-code them: https://github.com/node-ffi/node-ffi/issues/369

I know that issue is from `ffi`, not from `fastcall`, but there doesn't seem to be any reference or documentation about it in `fastcall` (or its issues).